js/bitmex.js

```diff
@@ -970,22 +970,24 @@ module.exports = class bitmex extends Exchange {
          */
         await this.loadMarkets ();
         const request = {
+            'currency': 'all',
             // 'start': 123,
         };
         //
         //     if (since !== undefined) {
         //         // date-based pagination not supported
         //     }
         //
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+            request['currency'] = code;
```

Comment:
If we decide to apply this filter here we should be using `currency['id']`instead but I'm getting an error every time I try to pass a currency 
![image](https://user-images.githubusercontent.com/43336371/181291313-247892c9-701d-4bba-afe6-75a8815443e7.png)

Did you manage to add a currency and get a valid response?

Reply:
You right, I will leave that part untouched, with just added `currency: all` as default for request.